### DISCUSSION TITLE
On Create Workflow: Update to Step 2 - FAILING

### DISCUSSION THREAD

**Discussion by colleenpridemore** [skills/.github#645] [Hello GitHub Actions]
I can't find exactly what is failing in the Create Welcome.yml Workflow.  I fear it might be my repo rulesets but I can't determine.  Little help?

**Comment by aflacake** [comment]
I had no problem and complete this pratice. 
Same failing status on step 2, to change file code ```welcome.yml```, try next **step 3** (don't forget refresh page)
copy and change the code at same file. Wait minutes and check your email github-actions[bot]
